packages/at_onboarding_cli/lib/src/util/home_directory_util.dart

```diff
@@ -50,4 +53,62 @@ class HomeDirectoryUtil {
             enrollmentId: enrollmentId),
         'hive');
   }
+
+  /// Generate a path like this:
```

Comment:
I really don't like that we are duplicating this code, again. I think instead we should move these functions to at_utils from at_cli_commons. Rather than delay this PR further, please create another ticket to take care of this.